packages/datetime/src/dateInput.tsx

```diff
@@ -112,6 +114,13 @@ export interface IDateInputProps extends IDatePickerBaseProps, IProps {
      * in the input field, pass `new Date(undefined)` to the value prop.
      */
     value?: Date;
+
+    /**
+     * Adds a time chooser to the bottom of the popover.
+     * Passed to the `DateTimePicker` component.
+     * @default undefined
```

Comment:
remove this. no need to declare `undefined` default. only use `@default` when it's explicitly set to a value (typically in `defaultProps`)